t/lib/ext-plugin.lua

```diff
@@ -367,6 +367,33 @@ function _M.go(case)
             local action = http_req_call_rewrite.End(builder)
             build_action(action, http_req_call_action.Rewrite)
 
+        elseif case.rewrite_resp_header == true then
+            local hdrs = {
```

Comment:
I think you can also add some unexpected headers and verify if they exist in the response headers.

Reply:
I have added two test cases.